examples/transport/1d-advection.jl

```diff
@@ -0,0 +1,94 @@
+# ~/~ begin <<docs/src/active-layer-transport.md#examples/transport/1d-advection.jl>>[init]
+#| creates: docs/src/_fig/1d-advection.svg
+#| collect: figures
+
+module Script
+    using CarboKitten
+    using CairoMakie
+
+    # ~/~ begin <<docs/src/active-layer-transport.md#transport-test-input>>[init]
+    transport_test_input(;
```

Comment:
All right, this function seems to be used at least three times.
So please replace by an import.